Source/Extensions/Blazorise.DataGrid/EventArguments/DataGridSortChangedEventArgs.cs

```diff
@@ -12,19 +14,37 @@ public class DataGridSortChangedEventArgs : EventArgs
     /// <summary>
     /// Default constructors.
     /// </summary>
-    /// <param name="fieldName">Column field name.</param>
+    /// <param name="fieldName">The column's sort field name.</param>
     /// <param name="sortDirection">Column sort direction.</param>
+    [Obsolete( message: "Please use the new constructor DataGridSortChangedEventArgs( string fieldName, string columnFieldName, SortDirection sortDirection ).", error: false )]
     public DataGridSortChangedEventArgs( string fieldName, SortDirection sortDirection )
+        : this( fieldName, fieldName, sortDirection )
+    {
+    }
+
+    /// <summary>
+    /// Default constructors.
+    /// </summary>
+    /// <param name="fieldName">The column's sort field name.</param>
+    /// <param name="columnFieldName">The column's field name.</param>
+    /// <param name="sortDirection">Column sort direction.</param>
+    public DataGridSortChangedEventArgs( string fieldName, string columnFieldName, SortDirection sortDirection )
     {
         FieldName = fieldName;
+        ColumnFieldName = columnFieldName;
         SortDirection = sortDirection;
     }
 
     /// <summary>
-    /// Gets the field name of the column that is being sorted.
+    /// Gets the field name of the column that is being sorted. The name is either <see cref="BaseDataGridColumn{TItem}.Field"/> or <see cref="DataGridColumn{TItem}.SortField"/> if specified.
     /// </summary>
     public string FieldName { get; }
 
+    /// <summary>
+    /// Gets the field name of the column that is being sorted. The name is always the same as in <see cref="BaseDataGridColumn{TItem}.Field"/>.
+    /// </summary>
+    public string ColumnFieldName { get; set; }
```

Comment:
What is the difference between `ColumnFieldName` and `FieldName`. Aren't they essentially the same thing? If `ColumnFieldName` is used for sorting then it should be named `SortByFieldName`.

Reply:
Is this a new api? If I remember correctly this api already exists so we shouldn't be making design decisions about jt now. We could however for 2.0
I think one represents the name of the column and the other the actual field used for sorting.

Reply:
Yes, it is a new API. It is marked as _add_ in the GitHub review mode

![image](https://github.com/Megabit/Blazorise/assets/900302/5ca8c7f3-9016-4ed9-93ad-831edd5b88dc)

Reply:
I was under the impression that already existed.... Strange...
Still, what we'd like to actually rename is the one that already exists. Naming seems fine since we can't rename the other one wouldn't you agree?

Reply:
`SortByFieldName`?

Reply:
No, I'm pretty sure that's the `FieldName` concern. In which case we could rename it in **2.0**.
Original current api in master : 
![image](https://github.com/Megabit/Blazorise/assets/22283161/d0a3834d-4edc-4ee0-a18a-9c0fb424d616)
![image](https://github.com/Megabit/Blazorise/assets/22283161/2b535032-d359-4b8f-ac66-e44e9e9ba9ca)


`ColumnFieldName` seems fine to me, as it wants to convey the meaning that's the original column name. We might want to reword the documentation so it's clearer.

Reply:
@stsrki can you recheck if at least the documentation change for these two makes it more clearer now?